projects/js-packages/number-formatters/package.json

```
@@ -14,28 +14,37 @@
 		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Number Formatters"
 	},
 	"dependencies": {
-		"@babel/runtime": "^7",
 		"@wordpress/date": "5.19.0",
-		"debug": "4.4.0"
+		"debug": "4.4.0",
+		"tslib": "2.5.0"
 	},
 	"devDependencies": {
 		"@babel/core": "7.26.10",
 		"@babel/preset-react": "7.26.3",
 		"@jest/globals": "29.4.3",
+		"@rollup/plugin-typescript": "12.1.0",
 		"@types/jest": "29.5.14",
 		"jest": "29.7.0",
 		"jest-environment-jsdom": "29.7.0",
+		"rollup": "3.29.5",
+		"rollup-plugin-dts": "6.1.1",
 		"typescript": "5.8.2"
 	},
+	"type": "module",
+	"main": "./dist/cjs/index.js",
+	"module": "./dist/mjs/index.js",
+	"types": "./dist/index.d.ts",
 	"exports": {
 		".": {
 			"jetpack:src": "./src/index.ts",
-			"types": "./dist/index.d.ts",
-			"default": "./dist/index.js"
+			"import": "./dist/mjs/index.js",
+			"require": "./dist/cjs/index.js",
+			"types": "./dist/index.d.ts"
```

Typescript calls for "types" to come before "import" and "require".
```suggestion
			"types": "./dist/index.d.ts",
			"import": "./dist/mjs/index.js",
			"require": "./dist/cjs/index.js"
```